config/deploy.rb

```diff
@@ -0,0 +1,47 @@
+# config valid only for current version of Capistrano
+lock '3.3.5'
+
+set :application, 'exercism'
+set :repo_url, 'git@github.com:code-R/exercism.io.git'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.